price-estimator/src/models/query.rs

```diff
@@ -52,6 +53,21 @@ pub enum EstimationTime {
     Timestamp(u64),
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum OrderbookKind {
+    // pricegraph instance with the original orders from the orderbook
+    Raw,
+    // pricegraph instance with the orders to which the rounding buffer has been applied
+    WithRoundingBuffer,
+}
```

Comment:
Wasn't orderbook kind being used for something entirely different in the past? I am guessing that we no longer use the outdated kind of orderbooks and now only use the streamed one.